newIDE/app/src/UI/Search/HighlightedText.js

```diff
@@ -39,6 +39,7 @@ const HighlightedText = ({
   matchesCoordinates: number[][],
   styleToApply: { backgroundColor?: string, color?: string },
 |}): React.Node[] => {
+  if (matchesCoordinates.length === 0) return [text];
```

Comment:
question: This matches everything? or nothing?

Reply:
It's a security in case the filtering actions after the search make the coordinates empty.

Reply:
Well, I'm checking

Reply:
Yep, that's it, if the coordinates are empty, we return a map on `returnText` that would be empty